indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -211,6 +211,29 @@
       (distinct (conj consortiums "GEOSS"))
       consortiums)))
 
+(defn- get-memory-statistics
```

Comment:
I think you should use the same function in cmr.common-app.services.jvm-info by making it public.

Reply:
Better yet I removed this function and the log-jvm-statstics function and added to the common-app/services/jvm-info.clj namespace the ability to add in an application name or any other string before the Maximum Memory message.